icons/notion/notion-plain.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
+    <path
```

Comment:
plain icons need to have only one path. See this tutorial for how to unite paths: https://github.com/devicons/devicon/issues/1306#issuecomment-1296014473

Reply:
Should I remove the last path in plain SVG?  @Snailedlt

Reply:
I don't understand via documentation

Reply:
@Yashh56 You'll need to download [inkscape](https://inkscape.org/), then follow the instructions in the comment I linked

Reply:
OK